config.properties.template

```diff
@@ -17,6 +17,10 @@ desired_catch_probability=0.4
 # set to -1 to not attempt catching at all
 desired_catch_probability_unwanted=0.0
 
+# number of nearest unused pokestops to randomly select
+# the next pokestop from (1/infinit)
+randomNextPokestop=1
```

Comment:
Can you rename this so this is named as others, eg `random_next_pokestop_selection`.